src/main/java/com/teamcautionrobotics/robot/Robot.java

```diff
@@ -25,131 +26,154 @@
  * project.
  */
 public class Robot extends TimedRobot {
-  Hatch hatch;
-  double armPower;
-  EnhancedJoystick driverLeft, driverRight;
-  Gamepad manipulator;
-  DriveBase driveBase;
-
-  double driveLeftCommand;
-  double driveRightCommand;
-
-  Timer timer;
-
-  boolean deployButtonPressed = false;
-
-  private static final String kDefaultAuto = "Default";
-  private static final String kCustomAuto = "My Auto";
-  private String m_autoSelected;
-  private final SendableChooser<String> m_chooser = new SendableChooser<>();
-
-  /**
-   * This function is run when the robot is first started up and should be
-   * used for any initialization code.
-   */
-  @Override
-  public void robotInit() {
-    hatch = new Hatch(2,1);
-    driverLeft = new EnhancedJoystick(0);
-    driverRight = new EnhancedJoystick(1);
-    manipulator = new Gamepad(2);
-    driveBase = new DriveBase(0, 1, 0, 1, 2, 3);
-
-    timer = new Timer();
-
-    m_chooser.setDefaultOption("Default Auto", kDefaultAuto);
-    m_chooser.addOption("My Auto", kCustomAuto);
-    SmartDashboard.putData("Auto choices", m_chooser);
-  }
-
-  /**
-   * This function is called every robot packet, no matter the mode. Use
-   * this for items like diagnostics that you want ran during disabled,
-   * autonomous, teleoperated and test.
-   *
-   * <p>This runs after the mode specific periodic functions, but before
-   * LiveWindow and SmartDashboard integrated updating.
-   */
-  @Override
-  public void robotPeriodic() {
-    armPower = .5 + .5 * manipulator.getAxis(Axis.LEFT_TRIGGER);
-    hatch.rotate(armPower);
-
-    //if B is pressed, deploy hatch pneumatics
+    VelcroHatch velcroHatch;
+    ExpanderHatch expanderHatch;
+    double armPower;
+    EnhancedJoystick driverLeft, driverRight;
+    Gamepad manipulator;
+    DriveBase driveBase;
+    AimingLights aimingLights;
+    double driveLeftCommand;
+    double driveRightCommand;
+
+    Timer timer;
+
+    boolean deployButtonPressed = false;
+
+    // These are for the Expander Hatch mechanism
+    boolean reacherButtonPressed = false;
+    boolean grabberButtonPressed = false;
+
+    private final boolean usingVelcroHatch = true;
+    private static final String kDefaultAuto = "Default";
+    private static final String kCustomAuto = "My Auto";
+    private String m_autoSelected;
+    private final SendableChooser<String> m_chooser = new SendableChooser<>();
```

Comment:
I thought we deleted this in #16.

Reply:
I was planning on fixing this when resolving merge conflicts.

Reply:
That seems good.